tasklist/app/controllers/tasks_controller.rb

```diff
@@ -17,16 +18,40 @@ def delete
 		redirect_to "/"
 	end
 
-	# def edit
-	# 	@task = Task.find(params[:id])
-	# 	render :new
-	# end
-
 	def create
 		Task.create(task_params[:task])
 		redirect_to "/"
 	end
 
+	def edit 
+		show
+	end
+
+	def update
+		show
+
+		@task.update(task_params[:task])
```

Comment:
I like the use of spacing within your controller actions to make it clear what parts are the setup, the actual change/action, and the result returned to the browser.